src/main/java/org/hiero/sketch/table/HashUtil.java

```diff
@@ -0,0 +1,116 @@
+package org.hiero.sketch.table;
```

Comment:
Maybe we should create a lib folder/package for this kind of stuff.
How about org.hiero.lib?